config/database.yml.sample

```diff
@@ -4,16 +4,16 @@ default: &default
 
 development:
   <<: *default
-  database: rubymem-dev
+  database: rubymem_dev
```

Comment:
Is this change just about style? or it has other meaning?

Reply:
@bronzdoc Names with dashes won't work with Postgres 12. 

<img width="740" alt="Screen Shot 2019-10-28 at 3 54 20 PM" src="https://user-images.githubusercontent.com/17584/67713043-7504b280-f99b-11e9-84bf-8b2ab7e52956.png">

So, that's why underscores are needed.

Reply:
TIL! thanks for the explanation.